modules/instrument_list/php/instrument_list.class.inc

```diff
@@ -214,9 +214,28 @@ class Instrument_List extends \NDB_Menu_Filter
                 $ddeStatus = new \NDB_BVL_InstrumentStatus;
                 $ddeStatus->select($instrument['DDECommentID']);
 
+                $db          = \Database::singleton();
+                $sessionLang = $db->pselectOne(
+                    'SELECT language_code
+                     FROM session 
+                     WHERE ID=:sid',
+                    array('sid' => $_REQUEST['sessionID'])
+                );
+                $altName     = $db->pselectOne(
+                    'SELECT Full_name_alternative
+                     FROM test_names_dictionary 
+                     WHERE Test_name=:tn AND language_code=:lang',
+                    array(
+                     'tn'   => $instrument['Test_name'],
+                     'lang' => $sessionLang,
+                    )
+                );
+                $fullName    = (!empty($altName)) ? $altName
+                    : $instrument['Full_name'];
+
                 $Ins = "instruments";
 
-                $ins = $instrument['Full_name'];
+                $ins = $fullName;
                 $this->tpl_data[$Ins][$x][$i]['fullName'] = $ins;
```

Comment:
Just a suggestion buy why not just have line 239 be:
`$this->tpl_data[$Ins][$x][$i]['fullName'] = $fullName;`
There are a few other lines in this class similar and where it seems like assigning a variable to $ins is pointless because the next few lines has it assigned to something else.

Reply:
It may have to do with our max line length of 80 chars... just my suspicion.

Reply:
@johnsaigle I didn't think of that and I guess then it will still be two lines.